tests/agent/conftest.py

```diff
@@ -44,7 +46,7 @@ def _mock_get_lastest_version_from_pypi(self) -> Version:
 @pytest.fixture
 def mock_context(mocker: MockerFixture) -> CloudDataContext:
     """Returns a `MagicMock` of a `CloudDataContext` for testing purposes."""
-    return mocker.MagicMock(autospec=CloudDataContext)
+    return mocker.MagicMock(autospec=CloudDataContext)  # type: ignore[no-any-return] #TODO: fix this
```

Comment:
Was also not sure how to resolve this `no-any-return` issue and why upgrading to GX Core 0.18.13 surfaced it.